src/Utils/decode-wa-message.ts

```diff
@@ -110,9 +110,9 @@ export function decodeMessageNode(stanza: BinaryNode, meId: string, meLid: strin
 		remoteJid: chatId,
 		fromMe,
 		id: msgId,
-		senderLid: stanza?.attrs?.sender_lid,
-		senderPn: stanza?.attrs?.sender_pn,
+		addressingMode: stanza?.attrs?.addressing_mode as 'pn' | 'lid',
```

Comment:
why adding addressingMode here? I think its better to keep it only on groupMetadata